lib/src/components/NxTabs/__tests__/NxStatefulTabs.test.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { mount } from 'enzyme';
+import NxStatefulTabs from '../stateful/NxStatefulTabs';
+import NxTabList from '../../NxTabList/NxTabList';
+import NxTab from '../../NxTab/NxTab';
+import NxTabPanel from '../../NxTabPanel/NxTabPanel';
+
+describe('NxStatefulTabs', function () {
+  it('selects the first tab automatically', function () {
```

Comment:
Should it be possible to specify a different initial tab than the first one? I'd think yes.

Reply:
At that point would you still want to use `NxStatefulTabs` though? If you were to allow someone to set a different initial tab it would need to be clear that changing the value of the initial tab wouldn't change the tab (ie on the re-render the internal state would be used instead). I worry that would confuse people trying to use this component.

Reply:
I came up with a way to handle this I think.

Reply:
> At that point would you still want to use NxStatefulTabs though? If you were to allow someone to set a different initial tab it would need to be clear that changing the value of the initial tab wouldn't change the tab (ie on the re-render the internal state would be used instead). I worry that would confuse people trying to use this component.

I think it would be an example of the [uncontrolled component](https://reactjs.org/docs/uncontrolled-components.html) pattern, which we already use in `NxStatefulTextInput`.

Reply:
Added `defaultActiveTab` prop
6fe2aeb